common/src/main/java/io/apicurio/registry/rest/v2/beans/VersionMetaData.java

```diff
@@ -64,7 +64,7 @@
      * (Required)
      * 
      */
-    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ssZ", timezone = "UTC")
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ss'Z'", timezone = "UTC")
```

Comment:
I guess this was generated by studio.
but we cannot change all of this things, it will break compatibility

Reply:
The Studio generator should be fixed - so I think this would be corrected if we did the generation again.